api/internal/states/install/states.go

```diff
@@ -37,6 +37,8 @@ const (
 	StateInfrastructureInstalling statemachine.State = "InfrastructureInstalling"
 	// StateInfrastructureInstallFailed is a final state of the install process when the infrastructure failed to isntall
 	StateInfrastructureInstallFailed statemachine.State = "InfrastructureInstallFailed"
+	// StateInfrastructureInstallSucceeded is the state of the install process when the infrastructure install has succeeded
+	StateInfrastructureInstallSucceeded statemachine.State = "InfrastructureInstallSucceeded"
```

Comment:
you need to add these to the valid state transitions object in https://github.com/replicatedhq/embedded-cluster/blob/main/api/controllers/kubernetes/install/statemachine.go#L9 and linux

Reply:
or perhaps this is a TODO